### PR TITLE
translation server and translation file

### PR DESCRIPTION
**Pull request by MashaelAlthawabi**

### CODE REVIEW

**Comment by yayobyte** [comment]
Hello Parcera.
I would love to have the service in a separated package. This is using the `package.json` file located in the root of the folder but it is better to have the concerns separated.

File: src/translations/index.ts [hunk 1]
```diff
@@ -0,0 +1,27 @@
+import express from 'express';
+import translations from './translation';
+
+const app = express();
+app.use(express.json());
+
+const PROT = 3000;
```
Comment (LilyaZakirova) [inline src/translations/index.ts:7]: `PORT` :)

**Comment by sceleskibsf** [comment]
What is the extension of `src/translations/translation` file?

**Comment by sceleskibsf** [comment]
Some formatting inconsistency in the index file - double check spaces and indentation.

**Comment by sceleskibsf** [comment]
Branch name containing author name? 👀

**Pull request closed without merging**